src/symbol/symbol_layout.js

```diff
@@ -168,6 +167,7 @@ export function performSymbolLayout(bucket: SymbolBucket,
             const variableTextAnchor = layout.get('text-variable-anchor');
             const radialOffset = layout.get('text-radial-offset').evaluate(feature, {});
 
+
```

Comment:
Can we somehow account for this no-extra-newlines preference in the linter? It would be great if the linter could eliminate the need for any style discussion.

Reply:
Captured in: https://github.com/mapbox/mapbox-gl-js/issues/8444